tables/hbo.tbl

```diff
@@ -0,0 +1,208 @@
+always מְּ 5-134-3  # mem with dagesh sheva
+always נּ 5-1345  # nun with dagesh
+always נְּ 5-1345-3  # nun with  dagesh sheva
+always פּ 5-1234  # pe with dagesh
+always פְּ 5-1234-3  # pe with dagesh sheva
+always קּ 5-12345  # qof with dagesh
+always קְּ 5-12345-3  # qof with dagesh sheva
+always שְׁ 146-3  #  Shin with dot and sheva
+always שְׂ 156-3  #  Sin sheva
+always שׁ 146  # shin with dot
+always שׂ 156  # sin
+always ת 1456  # tav
+always תּ 2345  # tav with dagesh
+always תְּ 2345-3  # tav with dagesh sheva
+always בּ 12  # Bet with dagesh
+always בְּ 12-3  # Bet with dagesh sheva
+always כּ 13  # KAF with dagesh 
+always כְּ 13-3  # KAF with dagesh  sheva
+always תּ 5-1456  # tav with daggesh
+always תְּ 5-1456-3  # tav with daggesh sheva
+
+# letter op code characters
+letter ֒ 46-234  # segol - isolated
+letter ֓ 46-146  # shalshelet
+letter ֔ 2  # zaqef qatan
+letter ֕ 46-1245  # zaqef gadol
+letter ֖ 46-2345  # tipeha
+letter ֗ 46-1235  # revia
+letter ֘ 46-145  # zarqa
+letter ֙ 46-34  # pashta
+letter ֚ 46-245  # yetiv
+letter ֛ 46-1236  # tevir
+letter ֜ 46-1  # geresh
+letter ֝ 46-16  # geresh muqdam
+letter ֞ 46-12  # gershayim
+letter ֟ 46-1345  # qarney para
+letter ֠ 46-2356  # telisha gedola
+letter ֡ 46-236  # pazer
+letter ֢ 46-23  # atnah hafukh
+letter ֣ 46-136  # munah
+letter ֤ 46-125  # mahapakh
+letter ֥ 46-134  # merkha
+letter ֦ 46-13  # merkha kefula
+letter ֧ 46-145  # darga
+letter ֨ 46-1234  # qadma
+letter ֩ 46-12345  # telisha qetana
+letter ֪ 46-13456  # yerah ben yomo
+letter ֫ 46-135  # ole
+letter ֬ 46-24  # iluy
+letter ֭ 46-24  # dehi
+letter ֮ 46-1356  # zinor
+letter ֯ 456-134  # masora circle
+letter ְ 3  # sheva
+letter ֱ 26  # hataf segol
+letter ֲ 25  # hataf patah
+letter ֳ 345  # hataf qamats
+letter ִ 24  # hiriq
+letter ֵ 34  # tsere
+letter ֶ 15  # segol
+letter ַ 14  # patah
+letter ָ 126  # qamats
+letter ֹ 456-246  # holam Malay
+letter ֺ 135  # holam haser
+letter ֻ 346  # qubuts
+letter ּ 5  # dagesh
+letter ֽ 4  # meteg
+letter ֿ 45  # rafe
+letter ׁ 456-4  # shin dot
+letter ׂ 456-45  # sin dot
+letter ׄ 456-5  # upper dot
+letter ׅ 456-56  # lower dot
+letter ׆ 456-1345  # nun hafukha
+letter ׇ 126  # qamats qatan
+letter א 1  # alef
+letter ב 1236  # bet
+letter ג 1245  # gimel
+letter ד 145  # dalet
+letter ה 125  # hey
+letter הּ 125  # hey - isolated
+letter ו 2456  # vav
+letter ז 1356  # zayin
+letter ח 1346  # het
+letter ט 2345  # tet
+letter י 245  # yod
+letter ך 16  # final kaf
+letter כ 16  # kaf
+letter ל 123  # lamed
+letter ם 134  # final mem
+letter מ 134  # mem
+letter ן 1345  # final nun
+letter נ 1345  # nun
+letter ס 234  # samekh
+letter ע 1246  # ayin
+letter ף 124  # final pe
+letter פ 1234  # Pe
+letter ץ 2346  # final tsadi
+letter צ 2346  # tsadi
+letter ק 12345  # qof
+letter ר 1235  # resh
+letter ש 456-146  # Shin
+letter װ 2456-2456  # double vav
+letter ױ 2456-245  # vav yod
+letter ײ 245-245  # double yod
+letter ׳ 456-126  # geresh
```

Comment:
I'm just curious about this choice of dots 456-126 for geresh, the single-apostrophe. 

I'm not familiar with the geresh, because it isn't used in Yiddish which is the only Jewish language I can answer questions about. I am familiar with gershayim (which I see on line 175 is dots 456), the double-quote mark that is used to indicate that you're reading an acronym , and I can infer that a geresh is the singular of gershayim. I googled it, and wikipedia tells me that the geresh is used in two different ways:
1) in modern Hebrew, to denote that a word is a loan word from another language and to indicate that the letter it's attached to is going to be pronounced using a sound that it doesn't usually make
2) as a cantillation mark. 

So I guess I'm curious why this little symbol got this combo of dots, and I'm curious if 456-126 is the ideal braille assignment for it (it seems like it might be confusing, given that both 456 and 126 have other meanings assigned in this table, but I am not here to second-guess you if this is really how this mark is usually transliterated). 

If it's being included in this table as a cantillation mark, maybe it should be grouped with the other cantillation marks, and maybe it should have the same 46 prefix as the other cantillation marks? I don't have an opinion about what it should be, I'm just curious why it's this.

Reply:
I also notice that apostrophe is defined as dot 3 on line 166 and I think if I were typing a geresh, I would reach for the apostrophe. I know they are technically not the same symbol, but because the geresh and apostrophe look the same, I would guess that they are often used interchangeably.  I noticed that the double-quote mark in Hebrew Braille according to Wikipedia is 356, and I wonder if that means that 356 would be a better choice for the gershayim.

Reply:
@Abby, Regarding accents and double quote, I realize you are thinking in terms of how the accent looks in print. braille readers don't necessarily think in terma of whether the braille symbol conforms to a similar print symbol. When the accent scheme was devised, the three of us made a conscious choice to represent the unofficially adopted accents with regard (as much as possible) to their name preceded by dots 46. So for example the zarkha was 46-z. We eventually ran out of letters and made choices for the rest. Ray has produced many thousands of pages of work using this system. His documents have been available at Bartimaeus (http://https://bartimaeus.us/brl-hot.html for a number of years and the accent system has also been accepted by Duxbury since 2014. It doesn't mean that things can't be changed but I would recommend that it be thought over carefully.

Reply:
thanks for explaining, @sjlarose ! I don't have opinions about what it should be, I'm just looking through this because I'm curious, and then I keep coming back to see if people have answered my questions (thank you!) and because I keep noticing things that seem like they might be errors, so then I ask more questions. I'm just glad you've got a system.